src/Microsoft.ML.PCA/WrappedPcaTransform.cs

```diff
@@ -104,7 +104,8 @@ public Reconciler(PipelineColumn weightColumn, int rank, Action<PcaTransform.Arg
             }
         }
 
-        /// <include file='doc.xml' path='doc/members/member[@name="Whitening"]/*'/>
+        /// <summary>Replace current vector with it's principal components. Can significantly reduce size of vector.</summary>
```

Comment:
>it's [](start = 49, length = 4)

You meant "its", not "it's".

[http://grammarist.com/spelling/its-its/](http://grammarist.com/spelling/its-its/)